### PR TITLE
Post,Put and & GetAll classrooms acceptance tests completed

### PR DESCRIPTION
**Pull request by tajwal**
Pust,Put and GetAll classrooms acceptance tests & RetrieveClassroomBYId service method completed.

### CODE REVIEW

File: OtripleS.Web.Api/Services/Classrooms/ClassroomService.cs [hunk 1]
```diff
@@ -18,14 +19,25 @@ public partial class ClassroomService : IClassroomService
...
         }
+                
+        public async ValueTask<Classroom> RetrieveClassroomById(Guid classroomId)
+        {
+            Classroom storageClassroom = 
+                await this.storageBroker.SelectClassroomByIdAsync(classroomId);
+            ValidateStorageClassroom(storageClassroom,classroomId);
+
+            return storageClassroom;
+        }
```
Comment (hassanhabib) [inline OtripleS.Web.Api/Services/Classrooms/ClassroomService.cs:40]: There are two tests for any of this.

**Comment by viralpandya** [comment]
You may want to separate PRs. One for Acceptance tests and other for service functions. And also RetrieveClassroomBYId is not developed with TDD (tests cases -> FAIL and PASS).

**Comment by tajwal** [comment]
> You may want to separate PRs. One for Acceptance tests and other for service functions. And also RetrieveClassroomBYId is not developed with TDD (tests cases -> FAIL and PASS).

Separate PR #75 is added for service.

File: OtripleS.Web.Api/Services/Classrooms/ClassroomService.cs (hunk 1 above)
Comment (tajwal) [reply OtripleS.Web.Api/Services/Classrooms/ClassroomService.cs]: UnitTests are added in separate PR #75

**Comment by hassanhabib** [comment]
Shutting this down for other similar work being completed.

**Pull request closed without merging**